includes/frontend/endpoints/class-mystyle-handoff.php

```diff
@@ -260,7 +261,9 @@ public function get_output() {
 						( MyStyle_Options::enable_alternate_design_complete_redirect() ) &&
 						( MyStyle_Options::get_alternate_design_complete_redirect_url() != null )
 				) {
-					$link = MyStyle_Options::build_alternate_design_complete_redirect_url($this->design);
+
+					// $link = MyStyle_Options::build_alternate_design_complete_redirect_url($this->design,$override);
+					$link = MyStyle_Design_Complete::get_redirect_url($this->design);
```

Comment:
@webbninja Please replace the if statement with simply `if ( null !== $link ) {`